integration/operator_test.go

```diff
@@ -334,3 +328,83 @@ func removeWorker(clusterName string) error {
 
 	return f.ApplyAWSConfigPatch(patch, clusterName)
 }
+
+func createHostPeerVPC() error {
+	log.Printf("Creating Host Peer VPC stack")
+
+	hostVPCStack := `AWSTemplateFormatVersion: 2010-09-09
+Description: CI Host Stack with Peering VPC and route tables
+Resources:
+  VPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.11.0.0/16
```

Comment:
For my understanding: is it reserved for testing? Or CF is smart enough to not create overlapping route tables?

Reply:
This is the CIDR used for testing. All the test peer accepter VPCs will have the same, they are isolated by definition. 

This prevents collisions in the routes used for peering, if we use the same VPC for all the builds (as we did before and as we do in production), then the routes used for each of the guest clusters  must have a different destination, we can't have two routes with the same CIDR destination in the same route table. We can solve this by assigning each guest a different, non-overlapping CIDR, but this didn't work for parallel builds because they all request a free CIDR at the same time, and they all get the same.

By using a new, separate peer accepter VPC for each build (and separate route tables associated to it just created for each build) we can make sure that the routes to the guest subnets have unique target CIDR, because both the accepter VPC and the route tables only live during the build. We can even assign all the guest VPCs the same CIDR, no need for IPAM at all.